crates/relayer/src/chain/ckb4ibc/message.rs

```diff
@@ -360,6 +384,7 @@ pub struct CkbTxInfo {
     pub envelope: Envelope,
     pub input_capacity: u64,
     pub event: Option<IbcEvent>,
+    pub commitment_path: String,
```

Comment:
Is this actually used?

Reply:
yes, it's used as the key of IBC messages cache, which I can look up for a quick search